### DISCUSSION TITLE
Latent multi-upscale workflow

### DISCUSSION THREAD

**Discussion by Ferniclestix** [Comfy-Org/ComfyUI#753] [Show and tell]
A workflow to split upscale into chunks, fine tune and sharpen then slap them all back together.
Requires: [WAS nodes](https://github.com/WASasquatch/was-node-suite-comfyui) , [Davemane](https://github.com/Davemane42/ComfyUI_Dave_CustomNode) 

[Heres](https://drive.google.com/file/d/1kEajfDowB-MPj8Vo51guOK8ahfPerGr2/view?usp=drive_link) a Txt file of the workflow.

Overview of the whole thing
<img width="1464" alt="multiupscaler4" src="https://github.com/comfyanonymous/ComfyUI/assets/128430314/badc096d-6877-43cd-be34-45ae9f9eab11">

Creation and initial generation -1
<img width="1354" alt="multiupscaler5" src="https://github.com/comfyanonymous/ComfyUI/assets/128430314/bb6e4f7e-a4f8-481a-80f4-f694b810f0a0">

Crop latent and upscale -2 
<img width="1334" alt="multiupscaler5-1" src="https://github.com/comfyanonymous/ComfyUI/assets/128430314/bfe89971-0e62-4989-9ebf-77cd238d9f5c">

Cropped multi sampling + multi latent composite  plus final output -3
<img width="1752" alt="multiupscaler6" src="https://github.com/comfyanonymous/ComfyUI/assets/128430314/973390d0-27a2-4b42-baae-9c907a1b0c72">

anyway, maybe someone will find it useful, its pretty good, made it for landscapes but can be easily modified for square images.
hopefully someone who codes can turn some of this into a single node lol.

**Comment by Saganaki22** [comment]
How would you go about turning this into a 1:1 instead of landscape Aspect ratio

**Reply by Ferniclestix** [reply]
duplicate and add two rows of images. ooorrr delete 4 and do some rewiring lol.

**Comment by rvion** [comment]
Hey,   would you be interested in converting it to cushy studio (based on comfy-ui) ? https://github.com/rvion/CushyStudio#readme 

Tool is still young / still alpha, but I'd love to get some great creators onboard to help grow the tool

**Reply by Ferniclestix** [reply]
I mainly do comfyUI workflow stuff because I have no ability to program, cushy looks very nice but big walls of text and me don't get along.

**Comment by Kademo15** [comment]
Would this basically be an ultimate sd upscale but with latent upscale and what would be the gains from using the latent space for an upscale.
